### DISCUSSION TITLE
Please add  Bybit & Bitverse wallets

### DISCUSSION THREAD

**Discussion by ghost** [rainbow-me/rainbowkit#1867] [Ideas]
Bybit: https://www.bybit.com/
Bitverse: https://www.bitverse.zone/

**Comment by 0xmkh** [comment]
@lyfmantle Added Bybit wallet in [this](https://github.com/rainbow-me/rainbowkit/pull/1874) PR

**Comment by 0xmkh** [comment]
PR for Bitverse wallet [here](https://github.com/rainbow-me/rainbowkit/pull/1875)
